table.go

```diff
@@ -76,6 +76,7 @@ type Table struct {
 	borders        Border
 	colSize        int
 	headerParams   []string
+	cellParams     []CellColor
```

Comment:
Could you please make new interface `CellParam` that just have method `param()` and type `[]CellParam` here. (FYI: struct CellColor should have `param()`)